ingenialink/utils/_utils.py

```diff
@@ -387,7 +387,6 @@ def convert_dtype_to_bytes(data, dtype):
         REG_DTYPE.U16: (2, False),
         REG_DTYPE.S16: (2, True),
         REG_DTYPE.U32: (4, False),
-        REG_DTYPE.S32: (4, True),
```

Comment:
Why do you delete it?

Reply:
By mistake, I will undo it.

Reply:
I did not delete the line, just the comma at the end as it is the final element of the dictionary.